### PR TITLE
snap_alias: using CmdRunner

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
snap_alias now using `CmdRunner` instead of `CmdModuleHelper`

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/modules/snap_alias.py

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]

File: plugins/modules/snap_alias.py [hunk 1]
```diff
@@ -106,25 +105,26 @@ class SnapAlias(CmdStateModuleHelper):
...
+        "name": cmd_runner_fmt.as_list(),
+        "alias": cmd_runner_fmt.as_list(),
+    }
 
     def _aliases(self):
         n = self.vars.name
         return {n: self._get_aliases_for(n)} if n else self._get_aliases()
 
     def __init_module__(self):
         self.vars.set("snap_aliases", self._aliases(), change=True, diff=True)
```
Comment (felixfontein) [inline plugins/modules/snap_alias.py:120]: I think this line indirectly needs `self.runner` to be present.

Comment (russoz) [reply plugins/modules/snap_alias.py]: done

**Review by felixfontein — APPROVED** [review]
As far as I can judge, this looks good. Also there are tests, so it should be fine :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**